### DISCUSSION TITLE
Shortcut key change and watermark color

### DISCUSSION THREAD

**Discussion by Zhangke977** [sumatrapdfreader/sumatrapdf#2879] [Q&A]
1.Whether to support custom shortcut keys？
How to define if you support custom shortcut keys?
2.The watermark color of some PDF documents is too dark, covering the original text.What do I need to do?

**Comment by GitHubRulesOK** [comment]
1) https://www.sumatrapdfreader.org/docs/Customizing-keyboard-shortcuts

2) check if you need to remove the watermark or see if its an overstrike problem with all viewers
